geode-serialization/src/main/java/org/apache/geode/internal/serialization/filter/SanctionedSerializablesFilterPattern.java

```diff
@@ -108,6 +108,22 @@ private static StringJoiner dependenciesPattern() {
         .add("sun.security.provider.certpath.SunCertPathBuilderException")
 
         // geode-modules
-        .add("org.apache.geode.modules.util.SessionCustomExpiry");
+        .add("org.apache.geode.modules.util.SessionCustomExpiry")
+        // geode-dunit
+        .add("org.apache.geode.management.JmxLocatorReconnectDistributedTest*")
+        .add("org.apache.geode.management.JmxServerReconnectDistributedTest*")
+        .add("org.apache.geode.management.internal.rest.ManagementRequestLoggingDistributedTest*")
+        .add("org.apache.geode.internal.metrics.MeterSubregistryReconnectDistributedTest*")
+        .add("org.apache.geode.internal.io.LocatorLauncherGlobalGlobalSerialFilterDistributedTest*")
+        .add("org.apache.geode.internal.cache.CompactOfflineDiskStoreDUnitTest*")
+        .add("org.apache.geode.internal.cache.ParallelDiskStoreRecoveryDUnitTest*")
+        .add("org.apache.geode.internal.cache.PartitionedRegionSingleHopDUnitTest*")
+        .add("org.apache.geode.internal.cache.BucketServerLocation66")
+        .add("org.apache.geode.internal.cache.ValidateOfflineDiskStoreDUnitTest*")
+        .add("org.apache.geode.logging.internal.LoggingWithReconnectDistributedTest*")
+        .add("org.apache.geode.distributed.HostedLocatorsDUnitTest*")
+        .add("org.apache.geode.management.internal.cli.commands.ShutdownCommandDUnitTest*")
+        .add("org.apache.geode.management.internal.cli.commands.ShutdownCommandOverHttpDUnitTest*")
+        .add("org.apache.geode.test.dunit.**");
```

Comment:
We need to figure out a different solution for this. We shouldn't add dunit tests to the actual product pattern for sanctioned serializables.

I suspect that these tests may be starting a LocatorLauncher which then rejects deserializing classes in geode-core/src/distributedTest.

Tests such as `CompactOfflineDiskStoreDUnitTest` should instead do one of the following:
1. set the global serial filter to `"*"` in the VM that invokes `LocatorLauncher.start`
2. change to setting a global serial filter by passing -Djdk.serialFilter to a new Locator JVM from StartLocatorCommand instead of configuring it from LocatorLauncher.start
3. add an undocumented System Property to skip the configuring of serial filter in LocatorLauncher.start and set that in ChildVM.main
4. use `serializable-object-filter` to add the test and all dunit rules as user classes to the Locator's global serial filter (which I think we currently have commented out in LocatorLauncher)

I think [1] is the better option. We have plenty of tests so we don't really need these random dunit tests doing anything with the global serial filter. What do you think?